### PR TITLE
Miu master 2.0

### PR DESCRIPTION
**Pull request by JimmyKuruvilla**
connects to https://github.com/RMSone/miu-insights/issues/1727

Sorry for the poor naming of branches:

Branch where I brought in in old features: miu-master-2.0
Branch I created so I could create a pull request (and I propose becomes new miu-master): miu-master-2. 

Check the `update to swagger 2.0` commit to ignore all the inspinia files.

### CODE REVIEW

File: miu.deploy/deploy_dev.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+##run gulp before running deploy
+
+set -eu
+cp conf/env-dev.js ../dist/env.js
+(cd .. && aws s3 sync dist/ s3://devapidocs.miuinsights.com --delete)
```
Comment (frosforever) [inline miu.deploy/deploy_dev.sh]: doesn't need a region?

File: src/main/html/index.html [hunk 2]
```diff
@@ -63,7 +96,7 @@
             hljs.highlightBlock(e)
           });
 
-          addApiKeyAuthorization();
+          // addApiKeyAuthorization();
```
Comment (frosforever) [inline src/main/html/index.html]: delete?

File: src/main/javascript/view/miu/modified [hunk 3]
```diff
@@ -0,0 +1 @@
+Added hiding to OperationView.js
```
Comment (frosforever) [inline src/main/javascript/view/miu/modified]: wha dis?

File: src/main/less/reset.less [hunk 4]
```diff
@@ -17,7 +17,7 @@ time, mark, audio, video {
     padding: 0;
     border: 0;
     font-size: 100%;
-    font: inherit;
+    font: inherit;
```
Comment (frosforever) [inline src/main/less/reset.less]: Ws

**Comment by frosforever** [comment]
👍 WNBC but admittedly, my vote shouldn't count for much here

File: src/main/javascript/view/miu/modified (hunk 3 above)
Comment (JimmyKuruvilla) [inline src/main/javascript/view/miu/modified]: mistake. removed

**Comment by 23will** [comment]
👍

**Pull request merged**